backend/src/main/java/com/redhat/cloud/notifications/routers/models/SettingsValuesByEventType.java

```diff
@@ -31,7 +31,7 @@ public static class ApplicationSettingsValue {
     public static class EventTypeSettingsValue {
         @JsonIgnore
         public String displayName;
-        public Map<EmailSubscriptionType, Boolean> emailSubscriptionTypes = new HashMap<>();
+        public Map<SubscriptionType, Boolean> emailSubscriptionTypes = new HashMap<>();
```

Comment:
`emailSubscriptionTypes` can't be renamed without requiring a frontend migration, which I don't want to do now.